### PR TITLE
40263 - Add script with go to review from product page when

### PR DESCRIPTION
**Pull request by kjugi**
DEV-40263

Add script to review button in product page

### CODE REVIEW

File: Magento_Review/web/js/go-to-review.babel.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+define(() => (config, element) => {
+  element.addEventListener('click', () => {
+    const element = document.getElementById(config.reviewId);
+
+    if (element) {
+      const parentElement = findAncestor(element, 'tab__content');
+      setActiveFormTab(parentElement, parentElement.dataset.content);
+    }
+  });
+
```
Comment (Januszpl) [inline Magento_Review/web/js/go-to-review.babel.js]: I would move content of anonymous to function to named function defined outside of the listener.

File: Magento_Review/web/js/go-to-review.babel.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        }
+      }
+      else {
+        if (tab.classList.contains('tab__title') && !tab.classList.contains('tab__title--active')) {
+          tab.classList.add('tab__title--active');
+        }
+
+        if (tab.classList.contains('tab__content') && !tab.classList.contains('tab__content--active')) {
+          tab.classList.add('tab__content--active');
+        }
```
Comment (Januszpl) [inline Magento_Review/web/js/go-to-review.babel.js]: Perhaps sth like that for adding and removing classes? (not necessary... just idea)
28-33 lines
```
addActiveClass(tab,'tab__title','tab__title--active');
addActiveClass(tab,'tab__content','tab__content--active');
```
```
function addActiveClass(el, selector, activeSelector) {
 if (el.classList.contains(selector) && !el.classList.contains(activeSelector) {
   el.classList.add(activeSelector)
}
```

File: Magento_Review/web/js/go-to-review.babel.js (hunk 1 above)
Comment (kjugi) [reply Magento_Review/web/js/go-to-review.babel.js]: moved, but don't know it's better than before

File: Magento_Review/web/js/go-to-review.babel.js (hunk 2 above)
Comment (kjugi) [reply Magento_Review/web/js/go-to-review.babel.js]: It's an ok idea but only for this case. I don't know if we have similar situations in other files but AFAIS there is not that much similar situations to this. I mean we are adding and removing classes but when we want to change it's often required to go deeper into the script. I want to get rid of that here now. Also, don't know if we need here such generall functions but I think it's topic to discuss

**Pull request merged**